crates/op-rbuilder/src/tests/vanilla/revert.rs

```diff
@@ -123,36 +123,50 @@ async fn revert_protection_bundle() -> eyre::Result<()> {
     assert!(block_generated.includes(*valid_bundle.tx_hash()));
 
     let bundle_opts = BundleOpts {
-        block_number_max: Some(4),
+        block_number_max: Some(5),
     };
 
-    let reverted_bundle = harness
+    let reverted_bundle_0 = harness
         .create_transaction()
         .with_revert()
+        .with_reverted_hash()
```

Comment:
This is a new helper function that adds into the reverted_txn_hashes field of the bundle the hash of the transaction. Thus, it includes that txn reverted on the block.